skiko/src/commonMain/cpp/common/include/mppinterop.h

```diff
@@ -22,6 +22,10 @@ namespace skikoMpp {
         int getClustersLength(SkTextBlob* instance);
         bool getClusters(SkTextBlob* instance, int* clusters);
     }
+
+    namespace finalizers {
+        void deleteString(void* instance);
```

Comment:
SkString*?

Reply:
this is reintepret_casted in deleteString body so that we don't need to be type-agnostic while disposing arbitrary pointers.

Reply:
Maybe it can be a something like `KInteropPointer` to mark that pointers are received from Kotlin side?

Reply:
Hmm, not sure if I understand. `deleteString` can only delete a string, so signature shall reflect that. When we want to cast that to generic `void ()(void*)` type - cast shall happen exactly at that moment.

Reply:
IMHO destructor function types should be `void (*)(KNativePointer)` or `void (*)(KinteropPointer)` to show, that they must be called on pointers received from Kotlin. We return `KNativePointer` from Cpp, so we probably should use this type everywhere we are receiving pointers